### DISCUSSION TITLE
Don't have infer button

### DISCUSSION THREAD

**Discussion by 257266** [voicepaw/so-vits-svc-fork#650] [Q&A]
Hello, how can i infer when so-vits-svc-fork v3.14.1 don't have infer button.
![Untitled](https://github.com/voicepaw/so-vits-svc-fork/assets/133726150/10d638df-850b-48bc-92db-d377eac59b30)

**Comment by njpwwelp** [comment]
> Hello, how can i infer when so-vits-svc-fork v3.14.1 don't have infer button. ![Untitled](https://user-images.githubusercontent.com/133726150/238512621-10d638df-850b-48bc-92db-d377eac59b30.jpg)

It seems you're in the same boat as me. I'm having the exact same problem. 
![Screenshot 2023-05-22 052914](https://github.com/voicepaw/so-vits-svc-fork/assets/113648917/40b63136-9c22-4e89-9cb9-c5a35d809e7b)
